### PR TITLE
[TASK] Add necessary packages to be able to work with react

### PR DESCRIPTION
**Pull request by phathoang**

### CODE REVIEW

File: Resources/Private/Javascript/client.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
```
Comment (MattiasNilsson) [inline Resources/Private/Javascript/client.js]: Replace " with '

File: Resources/Private/Javascript/client.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+import ReactDOM from "react-dom"
```
Comment (MattiasNilsson) [inline Resources/Private/Javascript/client.js]: Replace " with '

File: Resources/Private/Javascript/client.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+import Toolbars from "./components/Toolbars"
```
Comment (MattiasNilsson) [inline Resources/Private/Javascript/client.js]: Replace " with '

File: Resources/Private/Javascript/components/Toolbars.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
```
Comment (MattiasNilsson) [inline Resources/Private/Javascript/components/Toolbars.js]: Replace " with '

File: Resources/Private/Javascript/components/Toolbars.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+import TopBar from "./Toolbars/TopBar"
```
Comment (MattiasNilsson) [inline Resources/Private/Javascript/components/Toolbars.js]: Replace " with '

**Comment by phathoang** [comment]
Oops pushed too much

File: package.json [hunk 6]
```diff
@@ -9,16 +9,24 @@
     "add:resource:immutable": "NODE_ENV=production npm install && cp -a ./node_modules/immutable ./Resources/Public/Javascript",
     "build:extension": "npm run add:resource:ckeditor && npm run add:resource:toastr && npm run add:resource:immutable && npm run build:css",
     "build:css": "npm run lint:css --silent && node-sass Resources/Private/Styles/ -o Resources/Public/Styles/",
+    "build:js": "./node_modules/.bin/browserify Resources/Private/Javascript/client.js -o Resources/Public/Javascript/Bundle.js -t [ babelify --presets [ es2015 react ] ]",
```
Comment (MattiasNilsson) [inline package.json]: You can run only browserify as npm understands the path. Then also add a section about it to the readme.

**Comment by MattiasNilsson** [comment]
Really cool stuff! Just some minor nitpicks 👍

**Comment by phathoang** [comment]
@MattiasNilsson 
Ok pushed som fluid-template changes in 5cb7db415 so all we have will be just this https://www.dropbox.com/s/flpmsimo962zmxk/Sk%C3%A4rmklipp%202016-10-25%2013.45.13.png?dl=0

Can we merge this into react branch and then from that branch we can work on different components independently (maybe?)

**Comment by phathoang** [comment]
@MattiasNilsson  fixed your comments in c7682454

**Comment by phathoang** [comment]
@MattiasNilsson ok fixed the browserify path and updated readme.
I also added the bundle.js and build:js to build:extension.

**Pull request merged**